TIPS.md

```diff
@@ -0,0 +1,31 @@
+Instructions for play
+
+- While some parts of the mod are scripted, many parts (such as exploring and building/placing of invaders after setup) are not
+- When playing single player you can ignore the ready token and the "ready" text at the upper right (it's used for coordination among players)
+- Do not hit the rewind/fastforward (they do not work well with scripted mods)
+- To look at the reverse side of your spirit panel press Alt+Shift (default keybinding in TTS) and mouse over the spirit panel 
+- To zoom in on a card hold the Alt key (default keybinding in TTS) and mouse over the card/object
+
+Intermediate Tips:
+- Press numpad keys (as shown in the bottom right) to spawn new pieces
+  - Make sure numlock is enabled
+  - Note that these scripting keys can be rebound in the TTS settings
+- Lock pieces (right-click and select lock)
```

Comment:
hovering over the object and pressing `L` also works

Reply:
also Lock isn't directly on the context menu so it's Right Click -> Toggles -> Lock